data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg

```diff
@@ -445,11 +450,87 @@ Besides... I want my brother back."
             message= _ "I see them! There they are!"
         [/message]
 
+        [move_unit]
+            id=Muff Toras
+            to_x,to_y=9,4
+            force_scroll=yes
+        [/move_unit]
+
         [message]
             speaker=Muff Toras
             message= _ "Curses! If they had been an hour slower, our master’s reinforcements would already be here to meet us."
```

Comment:
Although this is not a part of the change, this line does seem a bit illogical. If the reinforcements were travelling towards there, it's unlikely that Arvith wouldn't get to fight them, as they'd be on his way, as far as I know. And IIRC there is nothing that even mentions these reinforcements later on.

Reply:
Good catch. I think this wording would be better, but it needs corresponding changes to the objectives and `time over` events (which both reference these reinforcements too) :

```
# po: There are no undead on the map yet, but Muff Toras summons some immediately after saying this
message= _ "Curses! If they had been a few hours slower, I might have been able to cover my tracks without arousing the elves’ suspicions. Still, no need for the disguise any more."
```

Reply:
I agree, it looks good. As for the turn limit, I guess unlimited could be fine. Or the time limit could remain the same, as they need to find Baran as soon as possible.